src/view.branch.js

```diff
@@ -0,0 +1,59 @@
+class BranchSwitch {
+  constructor($dom) {
+    this.$dom = $dom;
+    this.branches = [];
+    $dom.find('.octotree-views').on('click', '.octotree-header-branch', this._showPopUp.bind(this));
```

Comment:
Just minor question: how do people know the branch label is clickable ?

Reply:
well was thinking of what icon could be placed there to indicate it's clickable.